### ISSUE TITLE
wishlist: activity LED for each sample

### ISSUE THREAD

**Issue by adiknoth** [nicklan/drmr#3]
Hi!

I don't know if it makes sense, but I've tried drmr for the first time today. It was connected to an external MIDI keyboard, so I simply tried all the keys to trigger the samples.

If I would want to alter some settings (vol/pan), I would have to guess which key belongs to which sample. If you could provide a way to indicate an active sample, things would be considerably easier.

Maybe add an LED between the two knobs or change the box' background colour upon activity, maybe even some colour gradient depending on the currently played velocity.

Cheers

**Comment by nicklan** [comment]
Yeah, this is on my todo list.  Without some experimental lv2 features, there's no good way for the core of the plugin (the part that gets midi events) to tell the UI that something happened, but I think there's a way for me to request that the UI also be informed of midi events from the host.  I'm going to look into it shortly and if there is I'll add some sort of visual cue on activity for sure.

Thanks for the suggestion!

**Comment by nicklan** [comment]
This is now implemented in the lv2unstable branch.  

Without the lv2-atom extension this is totally impossible, so it can't be ported to the master branch.  You need some bleeding edge stuff for this to work, so if you're on stable everything you're probably out of luck.  You can get more info in the [README.md](https://github.com/nicklan/drmr/blob/lv2unstable/README.md) of that branch.  

At some point as these extensions stabilize and more hosts support them (which should hopefully be fairly soon), I'll merge these changes into master.
